UIKitDZ/UIKitDZ/PostScreen/PostModel/PostCell.swift

```diff
@@ -0,0 +1,260 @@
+        pageControll.pageIndicatorTintColor = .lightGray
+        pageControll.currentPageIndicatorTintColor = .black
+        return pageControll
+    }()
+
+    private let hearthButton: UIButton = {
+        let button = UIButton()
+        button.setImage(UIImage(named: "hearth"), for: .normal)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        return button
+    }()
+
+    private let commentButton: UIButton = {
+        let button = UIButton()
+        button.setImage(UIImage(named: "comment"), for: .normal)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        return button
+    }()
+
+    private let telegramButton: UIButton = {
+        let button = UIButton()
+        button.setImage(UIImage(named: "telegram"), for: .normal)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        return button
+    }()
+
+    private let saveButton: UIButton = {
+        let button = UIButton()
+        button.setImage(UIImage(named: "save"), for: .normal)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        return button
+    }()
+
+    private let likesLabel: UILabel = {
+        let label = UILabel()
+        label.textColor = .black
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.font = UIFont(name: "Verdana-Bold", size: 10)
+        return label
+    }()
+
+    private let comentLabel: UILabel = {
+        let label = UILabel()
+        label.textColor = .black
+        label.font = UIFont(name: "Verdana", size: 10)
+        label.lineBreakMode = .byWordWrapping
+        label.numberOfLines = 2
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+
+    private let imageComment: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = UIImage(named: "mainUserImage")
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        return imageView
+    }()
+
+    private let commentTextField: UITextField = {
+        let textField = UITextField()
+        textField.placeholder = "Комментировать ..."
+        textField.font = UIFont(name: "Verdana", size: 10)
+        textField.borderStyle = .none
+        textField.translatesAutoresizingMaskIntoConstraints = false
+        return textField
+    }()
+
+    private let timeLabel: UILabel = {
+        let label = UILabel()
+        label.textColor = .lightGray
+        label.font = UIFont(name: "Verdana", size: 10)
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+
+    // MARK: - Initializers
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        addViews()
+        setupCell()
+    }
+
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
```

Comment:
инит с fatal error лучше вроде заменять на такой
```suggestion
required init?(coder aDecoder: NSCoder) {
   super.init(coder: aDecoder)
}
```